optimizely/helpers/sdk_settings.py

```diff
@@ -49,6 +50,8 @@ def __init__(
             `send_event(type:, action:, identifiers:, data:)`
           fetch_segments_timeout: A fetch segment timeout in seconds (optional).
           odp_event_timeout: A send odp event timeout in seconds (optional).
+            Network timeout - how long to wait for odp platform to respond before failing.
```

Comment:
@jaeopt it's a clarification of what odp_event_timeout is.

I don't think our current definition is clear enough.

What do you think?

Reply:
@Mat001 it's good to have a clarification. We can try re-phrase it since "Network timeout - " looks dangling and not clear it's talking about both `fetch_segment_timeout` and `odp_event_timeout`.

Reply:
added better comment to both